pkg/controller/kogitoinfra/infinispan/resource.go

```diff
@@ -122,28 +122,6 @@ func newInfinispanResource(kogitoInfra *v1alpha1.KogitoInfra) *infinispan.Infini
 	return infinispanRes
 }
 
-// newInfinispanCredentials will generate custom credentials for the Infinispan Operator
-// Deprecated: not in use right now because of this issue: https://github.com/infinispan/infinispan-operator/issues/211
-func newInfinispanCredentials(kogitoInfra *v1alpha1.KogitoInfra) (*v1.Secret, error) {
```

Comment:
shouldn't we keep it here or open a jira to keep tracking it as it depends on infinispan issue to get fixed?

Reply:
I think opening a Jira for it is better as if we keep this function here golangcli-lint will give an error for unused func. If we do want to keep this here we can comment the whole function and uncomment and use whenever the infinispan issue is fixed

Reply:
@Kaitou786 try exporting the function and goLint will stop bothering. @spolti neither way we can retrieve this function back from the git history. I don't see any problems removing it if the "export" workaround won't work, or you guys think that would be better to get rid of it any way. 

I vote to get rid of it and open a JIRA to keep track of this bug on ISPN side with a link to the 0.10.0 tag, referencing the function.

Reply:
by Exporting the function golangci-lint does stop bothering, but still as @ricardozanini suggested it would be better to remove it for now and have a JIRA to track this, we can always retrieve this function from git history